run_fastsurfer.sh

```diff
@@ -448,9 +448,9 @@ if [ "$surf_only" == "0" ]; then
   echo "" |& tee -a $seg_log
 
   pushd $fastsurfercnndir
-  cmd="$python run_prediction.py --orig_name $t1 --pred_name $seg --conf_name $conformed_name $hires --ckpt_sag $weights_sag --ckpt_ax $weights_ax --ckpt_cor $weights_cor --batch_size $batch_size --cfg_cor $config_cor --cfg_ax $config_ax --cfg_sag $config_sag --run_viewagg_on $viewagg --device $device"
+  cmd="$python run_prediction.py --orig_name $t1 --pred_name $seg --conf_name $conformed_name --log_name $seg_log $hires --ckpt_sag $weights_sag --ckpt_ax $weights_ax --ckpt_cor $weights_cor --batch_size $batch_size --cfg_cor $config_cor --cfg_ax $config_ax --cfg_sag $config_sag --run_viewagg_on $viewagg --device $device"
   echo $cmd |& tee -a $seg_log
-  $cmd |& tee -a $seg_log
+  $cmd
   if [ ${PIPESTATUS[0]} -ne 0 ] ; then exit 1 ; fi
```

Comment:
does this pipestatus not also have to be modified, since the pipe/tee statement was changed?

Reply:
No, I think PIPESTATUS[0] will still get the exit code of the first command (index 0). This will be $cmd with or without a second command to which the output is piped (tee).